proof-producer/bin/proof-producer/CMakeLists.txt

```diff
@@ -65,6 +68,11 @@ function(setup_proof_generator_target)
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
     )
 
+    target_include_directories(${ARG_TARGET_NAME}-lib INTERFACE
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
```

Comment:
Copy-paste mistake: we do not generate any headers in build directory for this target.

Reply:
done